proxy-node-gateway/src/test/java/uk/gov/ida/notification/proxy/TranslatorProxyTest.java

```diff
@@ -57,16 +66,46 @@ public void shouldAcceptHubResponseTranslatorRequestAndReturnString() {
         );
         TranslatorProxy translatorProxy = new TranslatorProxy(
             jsonClient,
-            clientRule.baseUri()
+            UriBuilder.fromUri(clientRule.baseUri()).path("/translate-hub-response/happy").build()
         );
 
-        String samlResponse = translatorProxy.getTranslatedResponse(request);
+        String samlResponse = translatorProxy.getTranslatedResponse(request, "session-id");
 
         assertEquals("translated_saml_response_blob", samlResponse);
         Mockito.verify(jsonClient).post(
             request,
-            UriBuilder.fromUri(String.format("%s/translate-hub-response", clientRule.baseUri())).build(),
+            UriBuilder.fromUri(String.format("%s/translate-hub-response/happy", clientRule.baseUri())).build(),
             String.class
         );
     }
+
+    @Test
+    public void shouldThrowTranslatorResponseExceptionWhenErrorPostingToTranslator() {
+        HubResponseTranslatorRequest request = new HubResponseTranslatorRequest(
+            "hub_response",
+            "requestid",
+            "eidas_request_id",
+            "level_of_assurance",
+            UriBuilder.fromUri("http://connector.node").build(),
+            "connector_encryption_certificate"
+        );
+
+        TranslatorProxy translatorProxy = new TranslatorProxy(
+            jsonClient,
+            UriBuilder.fromUri(clientRule.baseUri()).path("/translate-hub-response/server-error").build()
+        );
+
+        try {
```

Comment:
Same as above

Reply:
@CharlesIC All done, if you could take another look?